wbooks/src/app/components/SerchIcon/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ReactElement } from 'react';
+import { View, Image } from 'react-native';
+import searchIcon from '@assets/General/ic_search.png';
+
+import styles from './styles';
+
+export default function SeachIcon(): ReactElement {
```

Comment:
This component is exactly the same as the NotificationIcon, you should make a generic component and send the image as a prop.